regression/testdata/chaos-test-input.yml

```diff
@@ -0,0 +1,79 @@
+organizations:
+  - name: org1
+    connProfilePath: ./connection-profile
+  - name: org2
+    connProfilePath: ./connection-profile
+  - name: org3
+    connProfilePath: ./connection-profile
+
+createChannel:
+  - channelPrefix: testorgschannel
+    sdk: cli
+    numChannels: 1
+    channelTxPath: ./channel-artifacts/
+    organizations: org1,org2,org3
+
+joinChannel:
+# joins all peers in listed organziations to all channels based on channelPrefix and numChannels
+  - channelPrefix: testorgschannel
+    numChannels: 1
+    organizations: org1,org2,org3
+
+anchorPeerUpdate:
+  - channelName: testorgschannel0
+    organizations: org1
+    anchorPeerUpdateTxPath: ./channel-artifacts/testorgschannel0org1anchor.tx
+  - channelName: testorgschannel0
+    organizations: org2
+    anchorPeerUpdateTxPath: ./channel-artifacts/testorgschannel0org2anchor.tx
+  - channelName: testorgschannel0
+    organizations: org3
+    anchorPeerUpdateTxPath: ./channel-artifacts/testorgschannel0org3anchor.tx
+
+installChaincode:
+# installs chaincode with specified name on all peers in listed organziations
+  - name: basic
+    version: 1.0
+    sdk: cli
+    targetPeers: peer0-org1,peer1-org1
+    path: chaincodes/chaos/node
+    organizations: org1
+    language: node
+    metadataPath: ""
+
+  - name: basic
+    version: 1.0
+    sdk: cli
+    targetPeers: peer0-org2,peer1-org2
+    path: chaincodes/chaos/node
+    organizations: org2
+    language: node
+    metadataPath: ""
+
+  - name: basic
+    version: 1.0
+    sdk: cli
+    targetPeers: peer0-org3,peer1-org3
+    path: chaincodes/chaos/node
+    organizations: org3
+    language: node
+    metadataPath: ""
```

Comment:
This will probably work, but I think the intended use is to put all the orgs in a single chaincode entry.
Having multiple chaincode entries is to support multiple different chaincodes or multiple versions.
Is there a reason you used one entry per org?

Reply:
Pretty sure that was how some of the others did it and when I tried to merge it into a single one I seem to remember it didn't work so I thought that was the design

Reply:
Would also allow for doing go and java chaincode at the same time in the future :-)

Reply:
The upshot is, merging into a single entry does not work when paths are used (so as to have auto generated connection profile names) and you need to split it out on a per org basis. If you are explicit in the connection profile names for each org, then you can have it merged into a single entity. 
2 options here then @denyeart which would you prefer ?

Reply:
I think this way is fine.